python/tests/integration/completions/test_chat_completions.py

```diff
@@ -65,10 +74,20 @@ def services() -> dict[str, tuple[ChatCompletionClientBase, type[PromptExecution
             default_headers={"Test-User-X-ID": "test"},
         ),
     )
+    azure_ai_inference_client = AzureAIInferenceChatCompletion(
+        client=ChatCompletionsClient(
```

Comment:
do we need to create a custom client for this? Why can't we just use AzureAIInferenceChatCompletion directly below? is that a matter of setup?

Reply:
Yes, because of a weird thing that they have where you need the API key in the header when using Azure openai endpoint.